README.md

```diff
@@ -22,6 +22,26 @@ The only supported format for images is SVG. The app works by changing the fill
 - Make sure to put `'` around the phrase and a `,` at the end
 - Chose "Commit directly to the `main` branch." and press `Commit changes`
 
+### Updating Colours
```

Comment:
Nice! This makes it easy for somene to drop in with a little or no tech knowledge to edit and alter colours.

I wonder if we should be encouraging them to use the branch & preview workflow though?

Reply:
It's quite a bit more complex to do branches, and if you have to make a second change via GitHub it can be confusing to be in the right place. It shouldn't deploy if anything breaks - but up to you if you know who will be maintaining this. Maybe add the Netlify deploy badge? I don't think I have access to do it.

Reply:
Fair point.

Badge: [![Netlify Status](https://api.netlify.com/api/v1/badges/906a8f00-ddb2-49d9-8b0a-37ec3c5b42c6/deploy-status)](https://app.netlify.com/sites/darts-colouring/deploys)

I will probably transfer the netlify to them at some point too, but use this for now.